plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -5789,6 +5789,194 @@ public void testDropTableDeleteData()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
```

Comment:
Please add a test with `Explain` for statistics. 
Wondering if you are missing about `TableStatisticsReader#makeTableStatistics`